src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/NewUser.vue

```diff
@@ -1,78 +1,108 @@
+            <v-col cols="12" md="6">
+              <v-text-field
+                v-model="email"
+                label="Email"
+                variant="outlined"
+                color="primary"
+                type="email"
+                :rules="[Rules.required(), Rules.email('Enter your email in the format \'name@email.com\'')]"
+                class="mt-5"
+              ></v-text-field>
+            </v-col>
+
+            <!-- Phone Field -->
+            <v-col cols="12" sm="6">
+              <v-text-field
+                v-model="phoneNumber"
+                label="Phone number"
+                variant="outlined"
+                color="primary"
+                :rules="[Rules.required(), Rules.phoneNumber()]"
+                @keypress="isNumber($event)"
+              ></v-text-field>
+            </v-col>
+
+            <!-- ECE Registration Section -->
+            <v-col cols="12">
+              <ECEHeader title="Your ECE registration" />
+              <p class="small mt-2">
+                To add an existing ECE certificate to your account to manage it online, you must enter the number now to link it to your account.
+              </p>
+              <p class="small mt-5">Do you have, or ever had, an ECE certificate in British Columbia?</p>
+
+              <!-- Radio Group for ECE Certificate -->
+              <v-radio-group v-model="eceCertificateStatus" :rules="[Rules.requiredRadio('Choose an option')]">
+                <v-radio label="Yes" :value="true"></v-radio>
+                <v-radio label="No" :value="false"></v-radio>
+              </v-radio-group>
+            </v-col>
+
+            <!-- ECE Registration Number -->
+            <v-col v-if="eceCertificateStatus === true" cols="12" sm="6">
+              <v-text-field
+                v-model="eceRegistrationNumber"
+                label="Your ECE Registration Number"
+                variant="outlined"
+                color="primary"
+                :rules="[Rules.required('Enter your ECE registration number')]"
+                @keypress="isNumber($event)"
+              />
+            </v-col>
+
+            <!-- Checkbox for Terms of Use -->
+            <v-col cols="12">
+              <v-checkbox v-model="hasAgreed" label="" color="primary" :rules="[Rules.hasCheckbox('You must read and accept the Terms of Use')]">
+                <template #label>
+                  <div>
+                    I have read and accept the
+                    <router-link to="/new-user/terms-of-use">Terms of Use</router-link>
+                  </div>
+                </template>
+              </v-checkbox>
+            </v-col>
+          </v-row>
+        </v-form>
+      </v-col>
+
+      <!-- Buttons -->
+      <v-col cols="12">
+        <v-row>
+          <v-btn rounded="lg" color="primary" class="mr-2" @click="submit">Save and continue</v-btn>
+          <v-btn rounded="lg" variant="outlined" @click="logout">Cancel</v-btn>
+        </v-row>
```

Comment:
It's good practice to always use v-row with v-col together. Noticed v-row without v-col for the buttons.

Reply:
Replaced with div!